mono/metadata/Makefile.am

```diff
@@ -64,10 +64,6 @@ endif
 
 glib_libs = $(top_builddir)/mono/eglib/libeglib.la
 
-if HOST_ANDROID
```

Comment:
I don't understand why we're getting rid of something that we apparently needed for android - and the commit message doesn't really explain it.

Reply:
Commit message says "portable", i.e. one thing for all platfroms.

There were three functions there: `creal, `cimag`, `cabs`.
`creal`, `cimag` are in mono-complex.h now, one line each, inlined, `cabs` is left in this file which is not deleted, but included for inlining. Another choice is no inlining and remove the if android and build the file for all platforms. But one line, should be inlined.

Reply:
Is not clear enough?